camayoc/tests/qpc/cli/test_scanjobs.py

```diff
@@ -73,7 +73,7 @@ def test_scanjob_with_multiple_sources(qpc_server_config, data_provider):
         should be available.
     """
     source_generator = data_provider.sources.defined_many(
-        {"type": Table.is_in(("network", "satellite", "vcenter"))}
+        {"type": Table.is_in(("network", "satellite"))}
```

Comment:
This only works on sources defined in a config. If you remove source of type `vcenter` from config, then this line can stay as-is.